packages/node_modules/@ciscospark/plugin-user/src/user.js

```diff
@@ -68,9 +68,7 @@ const User = SparkPlugin.extend({
     })
       .then((res) => {
         this.spark.credentials.set({supertoken: res.body.tokenData});
-        if (res.body.hasPassword) {
-          this.hasPassword = true;
-        }
+        this.hasPassword = true;
```

Comment:
Is this accurate? I thought the point of calling /activate was to determine if password was set.